### ISSUE TITLE
[Bug Report] <After Update Elvui & Merathilis is buggin>

### ISSUE THREAD

**Issue by csalkvist** [Merathilis/ElvUI_MerathilisUI#61]
### What is the issue you are having?

Played this morning with no issues. - Playiing Wotlk Classic
Came home and updated Merathillis via Curseforge (only addon with an update).
Now when I come ingame, I cant use keyboard commands. I cant see topbar - Cant really use anything. Cant change anything in elvui (See photos.
So I try and delete Cache. Nothing different.
I try and reinstall Merathillis. Nothing different.
I try and update Elvui via tukui.org. - There was a minor change. Still nothing different.
I try and remove merathillis and then reinstall it. Nothing different.

This issu
![Skærmbillede 2022-11-16 134244](https://user-images.githubusercontent.com/25384419/202186415-0ac24541-7d0f-4d22-83be-81a0a3886aa8.jpg)
![Skærmbillede 2022-11-16 135657](https://user-images.githubusercontent.com/25384419/202186538-bca7bdca-d83f-443d-8c74-a71b1af1497b.jpg)
![Skærmbillede 2022-11-16 135949](https://user-images.githubusercontent.com/25384419/202187195-5e54eee7-72e2-4d4d-8df1-c81bfcabf32b.jpg)

### What is the expected behavior?

Things should work

### What actually happened?

Think I did

### Errors

```shell
Message: ...s\ElvUI_MerathilisUI\Core\General\Functions\Core.lua:38: attempt to index global 'C_TooltipInfo' (a nil value)
Time: Wed Nov 16 13:54:40 2022
Count: 1
Stack: ...s\ElvUI_MerathilisUI\Core\General\Functions\Core.lua:38: attempt to index global 'C_TooltipInfo' (a nil value)
[string "@Interface\AddOns\ElvUI_MerathilisUI\Core\General\Functions\Core.lua"]:38: in main chunk

Locals: MER = <table> {
 SetDefaultModuleLibraries = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:363
 Media = <table> {
 }
 Enable = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:290
 RegisterChatCommand = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceConsole-3.0\AceConsole-3.0.lua:81
 EnableModule = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:328
 modules = <table> {
 }
 GetModule = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:206
 IterateEmbeds = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:438
 SecureHook = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceHook-3.0\AceHook-3.0.lua:335
 defaultModuleLibraries = <table> {
 }
 TimeLeft = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceTimer-3.0\AceTimer-3.0.lua:172
 UnregisterChatCommand = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceConsole-3.0\AceConsole-3.0.lua:107
 RawHookScript = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceHook-3.0\AceHook-3.0.lua:394
 ScheduleRepeatingTimer = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceTimer-3.0\AceTimer-3.0.lua:129
 CancelAllTimers = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceTimer-3.0\AceTimer-3.0.lua:160
 Printf = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceConsole-3.0\AceConsole-3.0.lua:65
 ClassIcons = <table> {
 }
 SetDefaultModulePrototype = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:405
 PLAYER_ENTERING_WORLD = <function> defined @Interface\AddOns\ElvUI_MerathilisUI\Init.lua:169
 Unhook = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceHook-3.0\AceHook-3.0.lua:417
 AddLib = <function> defined @Interface\AddOns\ElvUI_MerathilisUI\Init.lua:111
 name = "ElvUI_MerathilisUI"
 Initialize = <function> defined @Interface\AddOns\ElvUI_MerathilisUI\Init.lua:131
 IsEnabled = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:447
 orderedModules = <table> {
 }
 ScheduleTimer = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceTimer-3.0\AceTimer-3.0.lua:94
 DisableModule = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:346
 Libs = <table> {
 }
 Modules = <table> {
 }
 Title = "MerathilisUI "
 UnregisterMessage = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\CallbackHandler-1.0\CallbackHandler-1.0.lua:151
 RegisterMessage = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\CallbackHandler-1.0\CallbackHandler-1.0.lua:88
 GetArgs = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceConsole-3.0\AceConsole-3.0.lua:140
 Print = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceConsole-3.0\AceConsole-3.0.lua:50
 IsDevelop = false
 IsHooked = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceHook-3.0\AceHook-3.0.lua:492
 CancelTimer = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceTimer-3.0\AceTimer-3.0.lua:147
 hooks = <table> {
 }
 IsModule = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:464
 NewModule = <function> defined @Interface\AddOns\ElvUI_Libraries\Core\Ace3\AceAddon-3.0\AceAddon-3.0.lua:231
 SetDefaultModuleState = <function> defined @Interface\AddOns\ElvUI_Libraries\



This is first Error of 236 Error.
```

### ElvUI Changelog/Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/Merathilis/ElvUI_MerathilisUI/blob/development/CHANGELOG.md) and done the [Troubleshooting Steps](https://github.com/Merathilis/ElvUI_MerathilisUI/wiki/Troubleshooting).
- [X] I have disabled all addons except the AddOns called 'ElvUI' and 'ElvUI OptionsUI' and 'ElvUI_MerathilisUI' by using `/muidebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and/or MerathilisUI and another addon, make sure you include that information.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I gave you information above on which XPack i play Classic, WotLK or Retail.

**Comment by csalkvist** [comment]
Just now - I have reinstalled 5.37. 

Now its back to normal, so I suspect the 5.38 update to be bad?

**Comment by Merathilis** [comment]
I would not bad, i only just dont pretty much mainly care on classic versions. The error is happening because of a retail api, that just dont exist on classic

**Comment by Merathilis** [comment]
This should be fixed with 5.40
